nipoppy_cli/nipoppy/config/main.py

```diff
@@ -122,6 +160,14 @@ def save(self, fpath: str | Path, **kwargs):
             file.write(self.model_dump_json(**kwargs))
 
     @classmethod
-    def load(cls, path: str | Path) -> Self:
+    def load(cls, path: str | Path, apply_globals_replacement=True) -> Self:
```

Comment:
Should this change to `StrOrPathLike` type?

Reply:
That is added in #237 (which might already handle it), but otherwise I can't change it before the other PR is merged